src/main/java/core/basesyntax/reportservices/ReportToFileImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.reportservices;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+
+public class ReportToFileImpl implements ReportToFile {
```

Comment:
The class name `ReportToFileImpl` should be more specific to indicate the type of report it is implementing. Also, the class should be in a `service.impl` package if it's an implementation of a service interface.